### PR TITLE
Require benchmark to run in Travis

### PR DESCRIPTION
**Pull request by oliverlee**
This commit resolves https://github.com/pydy/pydy/issues/167.

### CODE REVIEW

File: bin/benchmark_pydy_code_gen.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 # external libraries
 from numpy import hstack, ones, pi, linspace, array, zeros, zeros_like
+import matplotlib
+matplotlib.use('Agg')
```
Comment (moorepants) [inline bin/benchmark_pydy_code_gen.py]: Should this be "if on Travis use agg"?

Comment (oliverlee) [inline bin/benchmark_pydy_code_gen.py]: The benchmark doesn't call plt.show() and just saves the file so it's fine
as is. Works on my machine as is.

We can add the Travis check but I don't know the variable to check.

Comment (moorepants) [inline bin/benchmark_pydy_code_gen.py]: But some people may not have the agg backend install on their on machine (not sure about this, maybe agg is always available on all platforms).

You can check the `TRAVIS` env variable to see if true.

Comment (oliverlee) [inline bin/benchmark_pydy_code_gen.py]: It looks like it is always available:
http://matplotlib.1069221.n5.nabble.com/Any-available-backend-for-non-root-users-td38528.html

Comment (moorepants) [inline bin/benchmark_pydy_code_gen.py]: ok

**Comment by moorepants** [comment]
+1, merge when tests pass.

File: .travis.yml [hunk 2]
```diff
@@ -52,3 +53,5 @@ script:
   - cd docs && make html && cd ..
   # Make sure the library installs.
   - python setup.py install
+  # Run benchmark
+  - python bin/benchmark_pydy_code_gen.py 4 100
```
Comment (moorepants) [inline .travis.yml]: You could make this: `1 10` so it runs faster. Any reason we need this to run with 4 links and 100 time steps?

Comment (oliverlee) [inline .travis.yml]: not really

**Pull request merged**